packages/coinstac-ui/app/render/state/ducks/consortia-page.js

```diff
@@ -1,20 +1,29 @@
-const SET_EXPANDED_RESULT = 'SET_EXPANDED_RESULT';
+const SET_EXPANDED_RESULTS = 'SET_EXPANDED_RESULT';
```

Comment:
It's nice for the variable name to match the action name. Makes debugging easier.